.ci/esy-build-steps.yml

```diff
@@ -10,6 +10,13 @@ steps:
 
   - script: 'esy test'
     displayName: 'esy test'
+  
+  - task: PublishTestResults@2
+    displayName: 'Publish JUnit file'
+    inputs:
```

Comment:
As we're building and testing on multiple platforms we might want to pass in the current platform and add this `buildPlatform: ${{ parameters.platform }}`

Reply:
I'd be in favor of that as well as making testRunTitle the platform as well because the current default of JUnit_junit.xml isn't great

Reply:
I'll do both then